README_ja.md

```diff
@@ -297,173 +297,174 @@ Flax、PyTorch、TensorFlowをcondaでインストールする方法は、それ
 
 🤗Transformersは現在、以下のアーキテクチャを提供しています（それぞれのハイレベルな要約は[こちら](https://huggingface.co/docs/transformers/model_summary)を参照してください）:
 
-1. **[ALBERT](https://huggingface.co/docs/transformers/model_doc/albert)** (Google Research and the Toyota Technological Institute at Chicago から) Zhenzhong Lan, Mingda Chen, Sebastian Goodman, Kevin Gimpel, Piyush Sharma, Radu Soricut から公開された研究論文: [ALBERT: A Lite BERT for Self-supervised Learning of Language Representations](https://arxiv.org/abs/1909.11942)
```

Comment:
@sgugger I'd like to merge this PR but for some reason the Japanese README's entire model list gets updated instead of two lines being added.

I've copied the content of this file from the Github UI and pasted it locally, ran `make fix-copies` but the same occurs.

Is it fine for me to merge?

Reply:
You have replaced those whole lines that contained Japanese by English-only content, so no it's not fine to merge as is. You should revert the changes to the Japanese README then run `make fixup` to add the new model.

Reply:
Thanks a lot, I didn't spot that.

Fixed now!